src/vanilla/Model/MethodGroupTS.cs

```diff
@@ -26,6 +26,74 @@ protected MethodGroupTS(string name): base(name)
         [JsonIgnore]
         public IEnumerable<MethodTS> MethodTemplateModels => Methods.Cast<MethodTS>();
 
+        public ISet<string> OperationModelNames
+        {
+            get
+            {
+                ISet<string> modelNames = new HashSet<string>();
+                while (true)
+                {
+                    int initialCount = modelNames.Count;
+                    foreach (Method method in Methods)
+                    {
+                        if (method.Body != null)
+                        {
+                            ModelsClosure(modelNames, method.Body.ModelType);
+                        }
+                        foreach (Response response in method.Responses.Values)
+                        {
+                            ModelsClosure(modelNames, response.Body);
```

Comment:
Can you rename `ModelsClosure()`? This method is really about collecting all of the model type names, but that isn't communicated to me with the name `ModelsClosure`.